sections/src/AdminPage.js

```diff
@@ -22,10 +23,31 @@ export default function AdminPage(): React.Node {
   const { config, currentUser } = useContext(StateContext);
 
   const [sheetURL, setSheetURL] = useState("");
+  const [studentFile, setStudentFile] = useState(null);
+  const [tutorFile, setTutorFile] = useState(null);
+  const [importStatus, setImportStatus] = useState("Import");
```

Comment:
If you're using some kind of enum here, please add flow typing so we know all the possible statuses. Look at `AttendanceStatus` in models.js to see one way of doing that.